src/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# code relating to tree structured stick breaking process
```

Comment:
Did you mean to include this in your PR?

Reply:
Removing it from the index.